.travis.yml

```diff
@@ -5,7 +5,7 @@ cache:
     - $HOME/.cache/pre-commit
 
 python:
-  - "3.5"
+  - "3.6"
```

Comment:
I put this specifically to P3.5, as it's the recommended supported Python version, which is then the most accurated one. Why are you changing it?

Reply:
Black is not working on 3.5, so I needed to change it

Reply:
OK, I didn't think about that... Well, I suppose there's no problem using P3.6...

Reply:
Let me check If I can make it use 3.6 only on linting :wink:

Reply:
Fixed!